.catalog-onboard-pipeline.yaml

```diff
@@ -6,7 +6,7 @@ offerings:
     catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
     offering_id: 90a14f92-4b86-4b27-acc1-dd6eefe0bfe9
     variations:
-      - name: standard
+      - name: fully-configurable
         mark_ready: true
         install_type: fullstack
         scc:
```

Comment:
Can you make same change as https://github.com/terraform-ibm-modules/terraform-ibm-scc/pull/262 here?